pages/blocks/[id].tsx

```diff
@@ -134,3 +30,23 @@ export default function BlockInformationPage() {
     </main>
   )
 }
+
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+
+export async function getStaticProps({ locale }) {
+  return {
+    props: {
+      ...(await serverSideTranslations(locale)),
+    },
+  }
+}
+
+// TODO: Revisit this:
+// Not certain this is how we want to do things but I wasn't able to support a purely
+// client side translation setup
+export async function getStaticPaths() {
```

Comment:
Seems like this might have to do `getServerSideProps`? I haven't looked into the details for how dynamic routing works with this